src/main/java/com/royalrangers/controller/achievement/TestForAdminController.java

```diff
@@ -0,0 +1,28 @@
+package com.royalrangers.controller.achievement;
+
+
+import com.royalrangers.bean.ResponseResult;
+import com.royalrangers.model.User;
+import com.royalrangers.repository.UserRepository;
+import com.royalrangers.service.achievement.TestForAdminService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TestForAdminController {
```

Comment:
Please rename controller and service to "AdminTestController" and "AdminTestService" or similar